pom.xml

```diff
@@ -29,11 +29,11 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.9</version>
+    <version>2.11</version>
   </parent>
 
   <artifactId>bouncycastle-api</artifactId>
-  <version>1.648.4-SNAPSHOT</version>
+  <version>2.16.0-SNAPSHOT</version>
```

Comment:
Why not simply `2.0-SNAPSHOT`? Is there some reason this is associated with a core version?